plugins/kuttl.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kuttl
+spec:
+  version: "v0.1.0"
+
+  shortDescription: Declaratively build, install, and run operators using KUTTL.
+  homepage: https://kudo.dev/
+  description: |
+    The KUbernetes Test TooL (KUTTL) is a highly productive test
+    toolkit for testing operators on Kubernetes.
+  caveats: |
+    See the documentation for more information: https://kudo.dev/docs/
```

Comment:
The linked page has no occurrence of `kuttl`.

Reply:
thanks @corneliusweig perhaps I jumped the gun.   `kuttl` is the testing portion of what was in `kudo`.   to be clear... `kudo` is still valuable for other purposes.   The docs for `kudo test`  under https://kudo.dev/docs/testing.html are all valid for `kuttl`.   We have the domain for kuttl.io an will be putting that up shortly.